backend/app/router/endpoints/quizes.py

```
@@ -0,0 +1,69 @@
+        (True, Question(question="What is 2+2?", answer="4")),
+        (False, MultiSelectQuestion(question="Which are fruits?", options=["Apple", "Car", "Banana", "Dog"], correct_answer=0)),
+    ],
+    score=0,
+    notes=["note1", "note2", "note3"],
+    type="short answer",
+    playermode="single"
+)
+quizes[sample_quiz.ID] = sample_quiz
+next_quiz_id = 2  # Next available ID
+
+class CreateQuizRequest(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)